sdk/deviceprovisioningservices/Azure.ResourceManager.DeviceProvisioningServices/samples/Generated/Samples/Sample_DeviceProvisioningServiceResource.cs

```diff
@@ -115,7 +115,7 @@ public async Task Update_DPSPatch()
         public async Task GetValidSkus_DPSGetValidSku()
         {
             // Generated from example definition: specification/deviceprovisioningservices/resource-manager/Microsoft.Devices/stable/2022-02-05/examples/DPSGetValidSku.json
-            // this example is just showing the usage of "IotDpsResource_ListValidSkus" operation, for the dependent resources, they will have to be created separately.
+            // this example is just showing the usage of "IotDpsResource_listValidSkus" operation, for the dependent resources, they will have to be created separately.
```

Comment:
it's `list`: https://github.com/Azure/azure-rest-api-specs/blob/49456eb149e3bb5d0528cbca714cfa1806415484/specification/deviceprovisioningservices/resource-manager/Microsoft.Devices/stable/2022-12-12/iotdps.json#L825